Rakefile

```diff
@@ -9,5 +9,5 @@ Yast::Tasks.configuration do |conf|
   conf.obs_sr_project = nil
   # TODO: improve it, at least do not get worse
   # TODO: remove condition when new packaging tasks are accepted to factory
-  conf.documentation_minimal = 50 if conf.respond_to?(:documentation_minimal=)
+  conf.documentation_minimal = 90 if conf.respond_to?(:documentation_minimal=)
```

Comment:
@jreidinger This is a good goal, but its fulfillment turns out problematic: currently we are at 75% and if you run `yard stats --list-undoc` you will see many undocumented `#label` and `#contents` methods. The problem is, YARD does not know classes from the outside, so you cannot even manually refer to the API like `# (see AbstractWidget#label)`.

What does work:
- This is hard to write (copy+paste to the rescue) but useful:
```rb
# @see http://www.rubydoc.info/github/yast/yast-yast2/CWM%2FAbstractWidget:label
```
- or this, not understood by Yard (no link) but counts as docs
```rb
# super CWM::AbstractWidget#contents
```
- or this, which is clearly cheating:
```rb
#.
```